### PR TITLE
Add a navigation bar

### PR DESCRIPTION
**Pull request by ConnorCimowsky**
screen shot 2015-06-29 at 12 51 38 pm

### CODE REVIEW

File: src-frontend/app/templates/components/navigation-bar.hbs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<nav class="navbar navbar-default">
+    <div class="container-fluid">
+        <div class="navbar-header">
+            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+                <span class="sr-only">Toggle navigation</span>
+                <span class="icon-bar"></span>
```
Comment (Kapin) [inline src-frontend/app/templates/components/navigation-bar.hbs]: Any reason for 3 of these?

File: src-frontend/app/templates/components/navigation-bar.hbs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+            </button>
+            {{#link-to 'index' class="navbar-brand"}}Sana Builder{{/link-to}}
```
Comment (Kapin) [inline src-frontend/app/templates/components/navigation-bar.hbs]: Sana Protocol Builder :)

File: src-frontend/app/templates/components/navigation-bar.hbs (hunk 1 above)
Comment (ConnorCimowsky) [inline src-frontend/app/templates/components/navigation-bar.hbs]: screen shot 2015-06-29 at 2 06 27 pm

**Comment by Kapin** [comment]
Only other comment is that we should have some way of logging out (and detecting who is logged in). Probably future tickets.

File: src-frontend/app/templates/components/navigation-bar.hbs (hunk 2 above)
Comment (ConnorCimowsky) [inline src-frontend/app/templates/components/navigation-bar.hbs]: I was trying to keep it short but okaaaaaay

File: src-frontend/app/templates/components/navigation-bar.hbs (hunk 1 above)
Comment (Ommy) [inline src-frontend/app/templates/components/navigation-bar.hbs]: Can't we use something like `glyphicon glyphicon-menu-hamburger` 
screen shot 2015-06-29 at 4 44 56 pm

Comment (ConnorCimowsky) [inline src-frontend/app/templates/components/navigation-bar.hbs]: Maybe, but I took this from the bootstrap website. Probably easier to just do it their way ¯\_(ツ)_/¯

**Comment by Trinovantes** [comment]
Man I didn't even review this while I was sick today :(

**Pull request merged**